.github/workflows/cronjobs.yaml

```diff
@@ -0,0 +1,17 @@
+name: GH Actions Cron Schedule
+on:
+  workflow_dispatch:
+  schedule:
+    # Every M-F at 12:00am run this job
+    - cron:  "0 0 * * 1-5"
+    
+jobs:
+  check-image-version:
+    uses: securesign/actions/.github/workflows/check-image-version.yaml@main
+    strategy:
+      matrix:
+        branch: [main, redhat-v1.3]
+    with:
+      branch: ${{ matrix.branch }}
```

Comment:
You can now specify the images you want to have managed. See: https://github.com/securesign/cosign/blob/72aac2145c2c68863c211747831674b40f7b46ca/.github/workflows/cronjobs.yaml#L16